keel-core/src/main/kotlin/com/netflix/spinnaker/keel/notifications/NotificationScope.kt

```diff
@@ -0,0 +1,5 @@
+package com.netflix.spinnaker.keel.notifications
+
+enum class NotificationScope {
+  RESOURCE
```

Comment:
Correct, I think we're safe here. We're only reading the time out of the database. That's a good point though. I tend to go with enum because it feels safer than strings, but it could end up being "too safe".